types/react/ts5.0/experimental.d.ts

```diff
@@ -123,7 +123,15 @@ declare module "." {
     ): void;
     function experimental_taintObjectReference(message: string | undefined, object: Reference): void;
 
+    export interface ViewTransitionInstance {
+        /**
+         * The {@link ViewTransitionProps name} that was used in the corresponding {@link ViewTransition} component or `"auto"` if the `name` prop was omitted.
+         */
+        name: string;
```

Comment:
I assumed every renderer would implement `name` since every renderer implements the prop. But maybe the props are actually renderer specific and we just need to [list their types on the Fiber](https://github.com/facebook/react/blob/056073de4c50b65807cd77ae6715c9ea8ee64277/packages/react-reconciler/src/ReactFiberViewTransitionComponent.js#L20)?